css/index.css

```diff
@@ -143,4 +252,67 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 1000px) {
+    header {
+        display: flex;
+        margin: 3% 0;
+    }
+    header nav {
+        font-size: 1.6rem;
+        display: flex;
+        justify-content: space-between; 
+        /* ^^^ WHY ISN'T THIS WORKING?!?!? ^^^ */
+    }
+    .logo {
+        display: flex;
+        order: -1;
+    }
+    .top-content .text-container {
+        font-size: 1.6rem;
+        margin-top: 3%;
+    }
+    .middle-content .boxes .box2 {
+        background: forestgreen;
+    }
+    .middle-content .boxes .box4 {
+        background: darkorchid;
+    }
+    .middle-content .boxes .box6 {
+        background: hotpink;
+    }
+    .middle-content .boxes .box8 {
+        background: indigo;
+    }
+    .middle-content .boxes .box10 {
+        background: lawngreen;
+    }
+    .bottom-content .text-container {
+        font-size: 1.6rem;
+    }
+}
+
+@media (max-width: 768px) {
+    
+    .middle-content .boxes .box1 {
+        background: teal;
+    }
+    .middle-content .boxes .box3 {
+        background: gold; 
+    }
+    .middle-content .boxes .box5 {
+        background: cadetblue;
+    }
+    .middle-content .boxes .box7 {
+        background: coral;
+    }
+    .middle-content .boxes .box9 {
+        background: crimson;
+    }
+}
+
+@media (max-width: 400px) {
+    remove(.middle-content .boxes .box10);
```

Comment:
We talked about this in our 1:1, but just as a refresher
```
.middle-content .boxes .box {
    display: none;
}
```
`!important` is important for sanity checks.